package.json

```diff
@@ -20,10 +20,11 @@
     "start": "node server/index.js",
     "test": "jest --coverage",
     "test:watch": "jest --watch --coverage",
-    "stress-test": "k6 run ./tests/k6-test/k6-test-script.js"
+    "load-test": "k6 run ./tests/k6-test/k6-test-script.js"
   },
   "dependencies": {
     "@babel/plugin-transform-runtime": "^7.11.5",
+    "axios": "^0.21.1",
```

Comment:
Looks good, but you should look at updating your version. There's a security problem with this version of Axios.

Reply:
Oh shoot really!? Thanks for the heads up, I'll go ahead and update Axios.

Reply:
I took the liberty of checking if there's a latest version for the Axios npm package I'm using, and it seems like the latest release is 0.21.1.